### ISSUE TITLE
[DUE BY 04/14/22 ] QA and test Wagtail 2.15.4 upgrade

### ISSUE THREAD

**Issue by johnnyporkchops** [fecgov/fec-cms#5132]
### Summary

 QA and testing after minor Wagtail upgrade to version 2.15.4

### Related issues/PR
Issue: https://github.com/fecgov/fec-cms/issues/4807
PR: https://github.com/fecgov/fec-cms/pull/5148

This PR branch is on dev:
https://dev.fec.gov/admin

 **This PR  is a minor version Wagtail upgrade, so we likely do not need to test every single template. However I am adding this list from the QA of the major version Wagtail upgrade to 2.0 as a reference. Please be as thorough as your bandwidth allows**. 
### Things to check
  - [ ] Does anything look unusual or awry in the Admin area? (You will not see images in the img admin but can upload)
  - [ ] Test uploading an image and adding to page
  - [ ] Test adding document to page
  - [ ] Test snippets
  - [ ] Test copy and paste into rich text editor blocks to make sure it retains links, glossary items, anchor links correctly
  - [ ] All templates working     
      - [ ] Alert for emergency use only 
      - [ ] Home page banner announcement
      - [ ] Record page 
      - [ ] Tips for treasurers page 
      - [ ] Press release page 
      - [ ] Custom page
      - [ ] Resource page
      - [ ] Reporting example page 
      - [ ] Reports landing page 
      - [ ] Meeting page 
      - [ ] Collection page 
      - [ ] Commissioner page 
      - [ ] Digest page 
      - [ ] Document feed page
      - [ ] Document page 
 

### New features: ( Don't have to test these, just FYI)
[Commenting](https://docs.wagtail.org/en/stable/editor_manual/editing_existing_pages.html#commenting):

<img width="1178" alt="Screen Shot 2022-04-05 at 11 01 45 PM" src="https://user-images.githubusercontent.com/5572856/161887951-d3cc7528-e857-4659-982b-841f672beb99.png">

<hr>

[Redirect export](https://docs.wagtail.org/en/stable/releases/2.13.html#redirect-export)
The redirects module now includes support for exporting the list of redirects to XLSX or CSV. This feature was developed by Martin Sandström.

<hr>

 [Typed table block](https://docs.wagtail.org/en/stable/releases/2.15.html#typed-table-block)
 Not yet implemented, but possibly coming soon for https://github.com/fecgov/fec-cms/issues/4258

**Comment by kathycarothers** [comment]
I tested out a number of pages in recreating their information on Feature. I did not encounter any problems. I also figured out the comment feature too. I like it. I tested the following pages:
 Custom page
 Resource page
 Reporting example page
 Meeting page
 Collection page
 Digest page
 Document feed page
 Document page
 
 @johnnyporkchops @djgarr @dorothyyeager

**Comment by dorothyyeager** [comment]
Thanks @kathycarothers! @johnnyporkchops I removed my name as a reviewer. I don't have bandwidth to test but it looks like Kathy has got this.

**Comment by kathycarothers** [comment]
I tested the following pages in Dev and had no problems:
I tested out a number of pages in recreating their information on Feature. I did not encounter any problems. I also figured out the comment feature too. I like it. I tested the following pages:
Custom page
Resource page
Reporting example page
Meeting page
Example page
Document feed page
Document page
Snippets
Record template

@johnnyporkchops

**Comment by prstoetzer** [comment]
I tested the Record page with a webinar article, which tend to be the most complicated articles we post in terms of Wagtail features used. Everything worked correctly.

I also tested creating an open meeting page [()).](https://dev.fec.gov/updates/april-28-2022-open-meeting/) I did find one issue - lacking an embed snippet for the informational message, I tried to publish the page without any snippet. The system does not allow that. Removing the snippet just resulted in it publishing with a blank message.
